conans/test/unittests/client/cmd/export/test_capture_export_scm_data.py

```diff
@@ -12,9 +12,10 @@
 from conans.test.utils.test_files import temp_folder
 from conans.test.utils.tools import create_local_git_repo, TestBufferConanOutput
 from conans.util.files import save
+from conans.paths.package_layouts.package_cache_layout import PackageCacheLayout
 
 
-@mock.patch("conans.client.cmd.export._replace_scm_data_in_conanfile", return_value=None)
+@mock.patch("conans.client.cmd.export._replace_scm_data_in_recipe", return_value=None)
```

Comment:
Now that I'm using the `package_layout` instead of the `destination_folder` directly, there is a call to `package_layout.conanfile()`. I could mock that call too or pass a mocked PackageLayout, but I think it is simpler to mock this function instead in this unittest.

Reply:
Yes, you are right, this is perfect as is.